docs/stake-pool-guide/node-installation-process.md

```diff
@@ -0,0 +1,239 @@
+```
+The Glorious Glasgow Haskell Compilation System, version 8.10.7
+```
+
+Check `cabal` version:
+```bash
+cabal --version
+```
+
+You should see something like this:
+
+```
+cabal-install version 3.6.2.0
+compiled using version 3.6.2.0 of the Cabal library
+```
+
+:::important
+Please confirm that the versions you have installed match the recommended versions above. If not, check if you have missed any of the previous steps.
+:::
+
+## Downloading & Compiling
+
+Let's create a working directory to store the source-code and builds for the components.
+
+```bash
+mkdir -p $HOME/cardano-src
+cd $HOME/cardano-src
+```
+Next, we will download, compile and install `libsodium`.
+
+```bash
+git clone https://github.com/input-output-hk/libsodium
+cd libsodium
+git checkout 66f017f1
+./autogen.sh
+./configure
+make
+sudo make install
+```
+
+Then we will add the following environment variables to your shell profile. E.G `$HOME/.zshrc` or `$HOME/.bashrc` depending on what shell application you are using. Add the following to the bottom of your shell profile/config file so that the compiler can be aware that `libsodium` is installed on your system.
+
+```bash
+export LD_LIBRARY_PATH="/usr/local/lib:$LD_LIBRARY_PATH"
+export PKG_CONFIG_PATH="/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH"
+```
+
+Once saved, we will then reload your shell profile to use the new variables. We can do that by typing `source $HOME/.bashrc` or `source $HOME/.zshrc` (***depending on the shell application you use***).
+
+We need to install Secp256k1 what is required for 1.35.0 cardano-node version
+
+Download and install libsecp256k1:
+```bash
+cd $HOME/cardano-src
+git clone https://github.com/bitcoin-core/secp256k1
+cd secp256k1
+git checkout ac83be33
+./autogen.sh
+./configure --enable-module-schnorrsig --enable-experimental
+make
+make check
+sudo make install
+```
+
+Now we are ready to download, compile and install `cardano-node` and `cardano-cli`. But first, we have to make sure we are back at the root of our working directory:
+
+```bash
+cd $HOME/cardano-src
+```
+
+Download the `cardano-node` repository:
+
+```bash
+git clone https://github.com/input-output-hk/cardano-node.git
+cd cardano-node
+git fetch --all --recurse-submodules --tags
+```
+Switch the repository to the latest tagged commit:
+
+```bash
+git checkout $(curl -s https://api.github.com/repos/input-output-hk/cardano-node/releases/latest | jq -r .tag_name)
+```
+
+:::important
+If upgrading an existing node, please ensure that you have read the [release notes on GitHub](https://github.com/input-output-hk/cardano-node/releases) for any changes.
+:::
+
+## Configuring the build options
+
+We explicitly use the `ghc` version that we installed earlier. This avoids defaulting to a system version of `ghc` that might be newer or older than the one you have installed.
+
+```bash
+cabal configure --with-compiler=ghc-8.10.7
+```
+
+If you are running non x86/x64 platform (eg. ARM) please install and configure LLVM with:
```

Comment:
I hadn't know this build target had become standardised, so I'm seeing it here for the first time myself: well done on including it 😎👍